app/src/main/java/com/willowtree/vocable/presets/Phrase.kt

```diff
@@ -12,6 +12,7 @@ sealed interface Phrase : Parcelable {
     val phraseId: String
     val sortOrder: Int
     val lastSpokenDate: Long?
+    val parentCategoryId: String?
```

Comment:
Just to make sure I understand, we needed this so that we can create the custom Phrase with the same category as the edited preset? Did this field need to be added to Custom and Preset?

Reply:
ah yea true. I was originally updating the custom phrase via a repo.addPhrase() and needed all of the properties but now that it's using the cool `@Update` tech we only need the properties being changed. removed!

```
storedPhrasesRepository.updatePhraseLocalizedUtterance(
    phraseId = phraseId,
    localizedUtterance = localizedUtterance,
)
```